### PR TITLE
Fixed text issues and image loading issues

### PR DESCRIPTION
**Pull request by ddc22**
#### Changes proposed in this Pull Request

* The initial image grid loading for the landing page had been slow. This was fixed by including a single image to be loaded.

| Before      | After |
| ----------- | ----------- |
| licecap  |  licecap-new    |

* Fixes to text suggested in the last call

#### Testing instructions
* Load the landing page and the thank you page and check that the lead time is changed to 3-4 business days
* The image should load once rather than in a grid

Related to #

### CODE REVIEW

File: client/my-sites/checkout/checkout-thank-you/difm/difm-lite-thank-you.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function DIFMLiteThankYou(): JSX.Element {
 
 					<h2 className="difm-lite-thank-you__header-text">
 						{ translate(
-							'Our Built By WordPress.com team will be in touch with you within 1-2 days when your site is ready to be transferred to your account and launched.'
+							'Our Built By WordPress.com team will be in touch with you when your site is ready to be transferred to your account and launched.'
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout-thank-you/difm/difm-lite-thank-you.tsx:25]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/marketing/do-it-for-me/landing.tsx [hunk 2]
```diff
@@ -93,7 +85,7 @@ export default function DoItForMeLandingPage(): JSX.Element {
 
 			<Paragraph>
 				{ translate(
-					'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 2-4 days your website will be ready to launch for just %(cost)s. The one-time fee includes a 4-page website and a 30-minute Quick Start Zoom orientation about your site when it’s complete.',
+					'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 2-4 business days your website will be ready to launch for just %(cost)s. The one-time fee includes a 4-page website and a 30-minute Quick Start Zoom orientation about your site when it’s complete.',
```
Comment (a8ci18n) [inline client/my-sites/marketing/do-it-for-me/landing.tsx:97]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Review by dzver — APPROVED** [review]
LGTM

**Comment by dzver** [comment]
Actually, I gave it thumbs up but then I noticed that it looks a bit funny on mobile. Is this how it's supposed to look?

Screenshot 2021-09-14 at 16 29 33

**Comment by ddc22** [comment]
Haha nice catch @dzver ! Will fix this! :)

**Comment by dzver** [comment]
LGTM on mobile as well

**Review by dzver — APPROVED** [review]

**Review by aneeshd16 — APPROVED** [review]
LGTM! Will you be deleting the old images as well?

**Comment by ddc22** [comment]
@aneeshd16 Actually a feature was requested to show a larger image on click so thought of leaving them for now

**Pull request merged**